docs/testrunner-toolkit/configuration/testcafe.md

```diff
@@ -65,13 +106,101 @@ __Example__:
   - name: "saucy test"
 ```
 
-### `match`
+### `browserName`
+
+__Description__: Name of desired browser. Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  browserName: "chrome"
+```
+
+### `src`
+
+__Description__: The explicit name, file glob, or location of the test files
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  src:
+    - "tests/test_file1.test.js"
+    - "tests/integrations"
+    - "*/*.test.js"
+```
+
+### `env`
+
+__Description__: Environment variables. It can be expanded.
+
+__Type__: *object*
+
+__Example__:
+```yaml
+  env:
+    hello: world
+    foo: $my_bar
+```
+
+### `screenshots`
+
+__Description__: Native screenshots settings for testcafe.
```

Comment:
How about `Screenshots settings for testcafe. See <link to testcafe>`

Reply:
good idea!